app/views/application/_nav.html.erb

```diff
@@ -7,6 +7,31 @@
       <strong><%= link_to 'Vesta', root_path %></strong>
     </div>
     <div id="responsive-menu">
+      <div class="top-bar-left">
+        <ul class="dropdown menu" data-dropdown-menu>
+          <% if current_user && policy(Draw).index? %>
```

Comment:
Ok, since all of these links depend on `current_user` we should extract that conditional to before the `ul.dropdown.menu` element.

Then, in the interest of time, we're going to build two menus, one for admins and one for students, and extract them as partials. We're going to put an example of this in the branch.